### PR TITLE
Fix Activity Page Contributors dropdown

### LINKED ISSUE #31261: [Bug] Activity Page Contributors -  type dropdown breaks non-English locale
### Description

When changing the Display Language to German in the Gitea Footer and trying to change the type in the contributors tab, from commits to any of the other ones, they don't get select and just end up empty.

### Gitea Version

1.22.0

### Can you reproduce the bug on the Gitea demo site?

Yes: https://gitea.com/Dominik_Saeume/nhplus/activity/contributors

### Screenshots

image

### How are you running Gitea?

https://demo.gitea.com/

### PR DESCRIPTION
**Pull request by wxiaoguang**
Fix #31261

### CODE REVIEW

File: routers/web/repo/contributors.go [hunk 1]
```diff
@@ -19,14 +19,8 @@ const (
...
 func Contributors(ctx *context.Context) {
 	ctx.Data["Title"] = ctx.Tr("repo.activity.navbar.contributors")
-
 	ctx.Data["PageIsActivity"] = true
 	ctx.Data["PageIsContributors"] = true
-
-	ctx.PageData["contributionType"] = "commits"
-
-	ctx.PageData["repoLink"] = ctx.Repo.RepoLink
-
```
Comment (wxiaoguang) [inline routers/web/repo/contributors.go:29]: Remove the abused `PageData`

File: web_src/js/components/RepoContributors.vue [hunk 2]
```diff
@@ -333,19 +335,17 @@ export default {
         <!-- Contribution type -->
         <div class="ui dropdown jump" id="repo-contributors">
           <div class="ui basic compact button">
-            <span class="text">
-              <span class="not-mobile">{{ locale.filterLabel }}&nbsp;</span><strong>{{ locale.contributionType[type] }}</strong>
-              <svg-icon name="octicon-triangle-down" :size="14"/>
-            </span>
+            <span class="not-mobile">{{ locale.filterLabel }}</span> <strong>{{ locale.contributionType[type] }}</strong>
+            <svg-icon name="octicon-triangle-down" :size="14"/>
```
Comment (wxiaoguang) [inline web_src/js/components/RepoContributors.vue:338]: Remove the abused `&nbsp;`

File: web_src/js/components/RepoContributors.vue [hunk 3]
```diff
@@ -333,19 +335,17 @@ export default {
...
           </div>
           <div class="menu">
-            <div :class="['item', {'active': type === 'commits'}]">
+            <div :class="['item', {'selected': type === 'commits'}]" data-value="commits">
```
Comment (wxiaoguang) [inline web_src/js/components/RepoContributors.vue:341]: Use `selected` to replace the abused `active`, and use `data-value` to provide the real "type" value for "change" event.

**Review by silverwind — APPROVED** [review]

**Review by yp05327 — APPROVED** [review]

**Comment by yp05327** [comment]
Maybe not related, but it seems not correct:
image

**Comment by wxiaoguang** [comment]
> Maybe not related, but it seems not correct:

Not related, and I guess even if it is "fixed", it's not worth to backport.

**Pull request merged**